RegisterPayments/function.json

```diff
@@ -0,0 +1,27 @@
+{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "route": "register-payments",
+      "methods": [
+        "post"
+      ]
+    },
+    {
+      "type": "http",
+      "direction": "out",
+      "name": "res"
+    },
+    {
+      "type": "blob",
+      "name": "rtDocument",
+      "path": "rt/{iuv}.xml",
```

Comment:
@gunzip How can I set this parameter or even others from the function? With [this](https://github.com/pagopa/pagopa-functions-rtmanager/pull/1/files#diff-72f69b451848ddc3d96ba0c5ef3d9b2bR290) I've ``` No value for named parameter 'iuv'.```

Reply:
fixed in 67bc7ae